### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by manjunjiao** [redradrat/kable#32]
https://github.com/redradrat/kable/blob/9e7205ad2327e3d3887177bfb9a70de8f6435f35/pkg/repositories/repositories.go#L341

Error while running kable terraform provider:

```panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xcb6e87]
goroutine 71 [running]:
github.com/redradrat/kable/pkg/repositories.unpointerify(...)
	github.com/redradrat/kable@v0.8.6/pkg/repositories/repositories.go:343
github.com/redradrat/kable/pkg/repositories.UpdateRegistry({0xc00023cbb0, 0x2, 0xc00028fbf0})
	github.com/redradrat/kable@v0.8.6/pkg/repositories/repositories.go:116 +0x27
github.com/redradrat/terraform-provider-kable/kable.glob..func2({0x112e178, 0xc00021be40}, 0x7f3b058b53c8, {0x80, 0xc000508000})
	github.com/redradrat/terraform-provider-kable/kable/concept_data_source.go:150 +0x10b
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc00021c8c0, {0x112e178, 0xc00021be40}, 0xc0001cefc0, {0x0, 0x0})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.8.0/helper/schema/resource.go:347 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).ReadDataApply(0xc00021c8c0, {0x112e178, 0xc00021be40}, 0xc000516e80, {0x0, 0x0})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.8.0/helper/schema/resource.go:558 +0xf7
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadDataSource(0xc0004d4018, {0x112e178, 0xc00021be40}, 0xc00002cf40)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.8.0/helper/schema/grpc_provider.go:1129 +0x389
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadDataSource(0xc000333ca0, {0x112e220, 0xc0000fc7b0}, 0xc000126050)
	github.com/hashicorp/terraform-plugin-go@v0.4.0/tfprotov5/tf5server/server.go:247 +0x1db
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadDataSource_Handler({0xf5d6e0, 0xc000333ca0}, {0x112e220, 0xc0000fc7b0}, 0xc00007fe00, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.4.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:416 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00037f880, {0x1140438, 0xc0000eb200}, 0xc0000290e0, 0xc0004f41b0, 0x19e8ed0, 0x0)
	google.golang.org/grpc@v1.38.0/server.go:1286 +0xc8f
google.golang.org/grpc.(*Server).handleStream(0xc00037f880, {0x1140438, 0xc0000eb200}, 0xc0000290e0, 0x0)
	google.golang.org/grpc@v1.38.0/server.go:1609 +0xa2a
google.golang.org/grpc.(*Server).serveStreams.func1.2()
	google.golang.org/grpc@v1.38.0/server.go:934 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.38.0/server.go:932 +0x294
Error: The terraform-provider-kable_v0.0.5 plugin crashed!```

**Comment by redradrat** [comment]
This is fixed since #33, right @mjiao? 

Please reopen if not.

**Comment by manjunjiao** [comment]
> This is fixed since #33, right @mjiao?
> 
> Please reopen if not.

That's right.
